src/npg_porch/db/data_access.py

```diff
@@ -307,7 +307,10 @@ async def get_long_running_tasks(self) -> list[TaskExpanded]:
                 lengths.setdefault(task.pipeline.name, []).append(
                     (task.updated - task.created).total_seconds()
                 )
-            elif task.status != TaskStateEnum.CANCELLED:
+            elif (
```

Comment:
This change might be right, but the code is difficult to read. A better version would be

```
for task in tasks:
            if task.status in [TaskStateEnum.CANCELLED, TaskStateEnum.FAILED]:
                next
            if task.status == TaskStateEnum.DONE:
                lengths.setdefault(task.pipeline.name, []).append(
                    (task.updated - task.created).total_seconds()
                )
            else:
                not_done.setdefault(task.pipeline.name, []).append(task)
```